### ISSUE TITLE
Composed modal is adding an attribute "role=region" but it does not have 'aria-label' or 'aria-labelledby'

### ISSUE THREAD

**Issue by mathioli** [carbon-design-system/carbon#4587]
## Brief description

When I used ```ComposedModal``` component, it creates a div element with role as region.

```<div class="bx--modal-container" role="region">...</div>```

Since there is a region role defined, the element must have 'aria-label' or aria-labelledby' which is not being added.

## Environment

> Browser
Chrome 78.0.3904.87

> What version of the Carbon Design System are you using?
v9.91.5

> What did you expect to happen?
The property aria-label or aria-labelledby is added

> What WCAG 2.1 checkpoint does the issue violate?
https://able.ibm.com/tools/token/c9f0aed4-dfcb-45be-87cc-548c74fbf546/aa74d680-3744-44b0-971c-e6a39e15ca53/archives/2019SeptDeploy//doc/w3/help/en-US/idhi_accessibility_check_g1158.html
IBM 2.4.1 Bypass Blocks

**Comment by emyarod** [comment]
`aria-label` is marked as a required prop if `hasScrollingContent` exists. are you passing in a value for `aria-label`?

**Comment by dakahn** [comment]
I'm going to close this one out. Can reopen if problem persists.
